### PR TITLE
Add kinematic parameters obtained from designers

### PR DESCRIPTION
**Pull request by 130s**
TODO: Add inertia information.

### CODE REVIEW

File: tetris_description/urdf/tetris.gazebo.xacro [hunk 1]
```diff
@@ -16,7 +16,7 @@
       <rightJoint>left_wheel_joint</rightJoint>
       <wheelSeparation>${wheel_separation}</wheelSeparation>
       <wheelDiameter>${wheel_diameter}</wheelDiameter>
-      <torque>5</torque>
+      <torque>0.8</torque>
```
Comment (130s) [inline tetris_description/urdf/tetris.gazebo.xacro]: Not sure if this the right place to specify the maximum torque.

Comment (534o) [inline tetris_description/urdf/tetris.gazebo.xacro]: this should be
```
+      <wheelTorque>0.8</wheelTorque>
+      <wheelAcceleration>0.05</wheelAcceleration>
```

Comment (534o) [inline tetris_description/urdf/tetris.gazebo.xacro]: I put acceleration limit, to prevent the flipover motion of the robot, due to sudden stop/start.

**Comment by 130s** [comment]
Updated according to https://github.com/tork-a/delivery/issues/183#issuecomment-80913668. @534o for the review.

File: tetris_description/urdf/tetris.urdf.xacro [hunk 2]
```diff
@@ -125,6 +128,7 @@
       <hardwareInterface>EffortJointInterface</hardwareInterface>
     </joint>
     <actuator name="right_wheel_motor">
+
```
Comment (534o) [inline tetris_description/urdf/tetris.urdf.xacro]: is might be typo

File: tetris_description/urdf/tetris.gazebo.xacro [hunk 3]
```diff
@@ -16,7 +16,8 @@
       <rightJoint>left_wheel_joint</rightJoint>
       <wheelSeparation>${wheel_separation}</wheelSeparation>
       <wheelDiameter>${wheel_diameter}</wheelDiameter>
-      <torque>5</torque>
+      <wheelTorque>0.8</wheelTorque>
+      <wheelAcceleration>0.05</wheelAcceleration>
```
Comment (534o) [inline tetris_description/urdf/tetris.gazebo.xacro]: this need to be confimed by manufactular, if possible

Comment (130s) [inline tetris_description/urdf/tetris.gazebo.xacro]: While we play with this value in a trial to find a good reasonable value, this will be asked.

**Pull request merged**